model/src/w3swldmd.F90

```diff
@@ -362,7 +406,20 @@ SUBROUTINE W3SWL6 (A, CG, WN, S, D)
 !/
       END SUBROUTINE W3SWL6
 !/ ------------------------------------------------------------------- /
-!/
+!
+!> @private
```

Comment:
Probably you don't want to use the private tag. We have the EXTRACT_PRIVATE setting on, and, in general for science codes, we want documentation for all functions, even functions with private fortran scope.

Reply:
OK, I agree.  Will make this change.